core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -97,13 +108,27 @@ bool eventSetsEqual(const EventSet& l, const EventSet& r) {
          l.end_fence == r.end_fence && l.sync_dual_view == r.sync_dual_view &&
          l.modify_dual_view == r.modify_dual_view &&
          l.declare_metadata == r.declare_metadata &&
+         l.request_responses == r.request_responses &&
+         l.transmit_actions == r.transmit_actions &&
          l.declare_input_type == r.declare_input_type &&
          l.declare_output_type == r.declare_output_type &&
          l.end_tuning_context == r.end_tuning_context &&
          l.begin_tuning_context == r.begin_tuning_context &&
          l.request_output_values == r.request_output_values &&
          l.declare_optimization_goal == r.declare_optimization_goal;
 }
+template <typename Callback, typename BooleanConstant, typename... Args>
+inline void call_kokkos_callback(const Callback callback, BooleanConstant,
+                                 Args... args) {
```

Comment:
Look at std::invoke

Reply:
and rename `call_ -> invoke_`

Reply:
46e4042

Note that because ours are all function pointers, I'm not worrying about specializations, I'm just writing the version for function pointers